### PR TITLE
Goofmos 1.5: Balance and Gauges

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
del: Freon and Water Vapor no longer destroy themselves in the air. Use the Air Alarms to turn on the scrubbers via either the Contaminated modes or manually hitting the buttons ala plasma.
experimental: Freon now starts off at 2.7K again, which means there is way more of it in the canisters again. Canisters start with less pressure to balance this out.
add: Added a fancy pressure gauge to canisters. Check it out!
/:cl:
## **#20301 fixed air alarms being unable to scrub Freon or Water Vapor**

This should make freon floods matter a bit more, as right now they just kinda dissipate and can't be used to REALLY flood. Balance testing will show.

Also some sanity checking for reagents on datums.
### Added a FANCY ASS PRESSURE GAUGE, CHECK THE WEBM BELOW

https://www.dropbox.com/s/5ffatp07wlwzzvd/gas_tank.webm?dl=0

### CODE REVIEW

**Comment by PKPenguin321** [comment]
water vapor no longer evaporates?

the freon i get but water vapor is kinda ehhhh

**Comment by Durkel** [comment]
If there's no holding tank tank for freon/water vapor it's just going to sit and clog up the mix tank, no?

**Comment by Iamgoofball** [comment]
Yeah, but you can siphon it out with a on the scrubber/mix tank line

**Comment by Iamgoofball** [comment]
https://www.dropbox.com/s/5ffatp07wlwzzvd/gas_tank.webm?dl=0

**Comment by PKPenguin321** [comment]
ooh, fancy! one thing I would add: make the tip of the little stick that moves along the gauge have a different color at the end so it's easier to see on the black background

File: tgui/src/tgui.ract [hunk 1]
```diff
@@ -17,6 +17,7 @@ component.exports = {
     'ui-section': require('components/section'),
     'ui-subdisplay': require('components/subdisplay'),
     'ui-tabs': require('components/tabs'),
+    'ui-gauge': require('../node_modules/ractive-gauge/dist/ractive-gauge'),
```
Comment (optimumtact) [inline tgui/src/tgui.ract]: This doesn't look right

File: tgui/package.json [hunk 2]
```diff
@@ -43,6 +43,7 @@
     "ractive-componentify": "0.2.4",
     "ractive-events-keys": "0.2.1",
     "ractive-transitions-fade": "0.3.1",
+	"ractive-gauge": "0.0.10",
```
Comment (optimumtact) [inline tgui/package.json]: can you fix the spacing on this? it's triggering me

**Comment by MrStonedOne** [comment]
Make the pressure needle on the gauge not be black on a black background you goof.

Nobody can see it.

**Comment by Iamgoofball** [comment]
I'll make a PR to the upstream.

File: tgui/package.json (hunk 2 above)
Comment (Iamgoofball) [inline tgui/package.json]: It shows up fine locally, idk what broke

File: tgui/src/tgui.ract (hunk 1 above)
Comment (Iamgoofball) [inline tgui/src/tgui.ract]: it just works

I'm not a JavaScript guy

**Comment by MrStonedOne** [comment]
at the end of this [line](https://github.com/konsumer/ractive-gauge/blob/gh-pages/dist/ractive-gauge.js#L6), change `stroke:#000` to `stroke:{{line_color}}` then add `line_color: #000` to [here](https://github.com/konsumer/ractive-gauge/blob/gh-pages/dist/ractive-gauge.js#L12)(don't forget to add a , after the value line) then you can just add line_color="blah" to `<ui-gauge>`

File: tgui/package.json (hunk 2 above)
Comment (optimumtact) [inline tgui/package.json]: spaces vs tabs, that file uses spaces, so you should too

Comment (optimumtact) [inline tgui/package.json]: that's not the node standard

Comment (Iamgoofball) [inline tgui/package.json]: @Cheridan whats the codebase standard

**Comment by Supermichael777** [comment]
that needle is too dark

**Comment by pudl** [comment]
@Iamgoofball I promise I'm holding u down no matter what... fuck the world and what they think\feel I'm with u for life, ur the reason I map.........
I just wish u could love me the way I love u big bro... ur my idol for life and I hope ur sorry for not liking my AI change, not knowing how much I love u...

**Comment by Iamgoofball** [comment]
@MrStonedOne PR'd to upstream, repo owner asked that I make changes in the svg itself previously though.
https://github.com/konsumer/ractive-gauge/pull/2

**Comment by MrStonedOne** [comment]
You can't make that kind of change to the svg directly.

File: tgui/package.json (hunk 2 above)
Comment (MrStonedOne) [inline tgui/package.json]: The code base standard is tabs

**Comment by Iamgoofball** [comment]
fuck

File: tgui/package.json (hunk 2 above)
Comment (optimumtact) [inline tgui/package.json]: IT'S NOT DM CODE, IT'S A JS CONFIG FILE FOR NODE

Comment (MrStonedOne) [inline tgui/package.json]: Its, its only config. Y u had 2 b mad?

**Comment by KorPhaeron** [comment]
Stale

**Pull request closed without merging**